itests/src/test/java/fi/metatavu/edelphi/test/ui/base/index/ProfileTestsBase.java

```diff
@@ -18,73 +15,31 @@ public void testLoginRequired() {
   }
   
   @Test
-  public void testSubscriptionLevelNone() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    login(ADMIN_EMAIL);
-    navigate(PROFILE_PAGE);
-    waitAndAssertText("#profileSubscriptionLevelBlockContent p:first-child", "You do not have an active subscription");
-  }
-  
-  @Test
-  public void testSubscriptionLevelEdu() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    updateUserSubscription(1l, "EDU", toDate(2016, 1, 1), toDate(2016, 10, 10));
-    login(ADMIN_EMAIL);
-    navigate(PROFILE_PAGE);
-    assertSubscriptionLevelTexts("Educational", 1, 25); 
-  }
-  
-  @Test
   public void testSubscriptionLevelBasic() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
     updateUserSubscription(1l, "BASIC", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
     assertSubscriptionLevelTexts("Basic", 1, 50);
   }
   
   @Test
-  public void testSubscriptionLevelPro() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    updateUserSubscription(1l, "PRO", toDate(2016, 1, 1), toDate(2016, 10, 10));
+  public void testSubscriptionLevelPlus() {
+    updateUserSubscription(1l, "PLUS", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
-    assertSubscriptionLevelTexts("Professional", 2, 100);
+    assertSubscriptionLevelTexts("Plus", 2, 100);
   }
-  
+
   @Test
-  public void testSubscriptionLevelUnlimited() {
-    if (SUBSCRIPTION_LEVELS_DISABLED) {
-      return;  
-    }
-    
-    updateUserSubscription(1l, "UNLIMITED", toDate(2016, 1, 1), toDate(2016, 10, 10));
+  public void testSubscriptionLevelPremium() {
+    updateUserSubscription(1l, "PREMIUM", toDate(2016, 1, 1), toDate(2016, 10, 10));
     login(ADMIN_EMAIL);
     navigate(PROFILE_PAGE);
-    waitAndAssertText(".profileSubscriptionLeveText", String.format("Your current subscription level is \"%s\".", "Unlimited"));
-    assertNotPresent(".profileSubscriptionFeaturesLabel", ".profileSubscriptionPanels", ".profileSubscriptionPanelists");
-    waitAndAssertText(".profileSubscriptionEnds", "your subscription ends Oct 10, 2016");
-    waitAndAssertText(".profileChangeSubscriptionText", "You may change or continue your subscription by clicking");
-    waitAndAssertText(".profileChangeSubscriptionLink", "here"); 
+    assertSubscriptionLevelTexts("Premium", 2, 100);
   }
-
+  
   private void assertSubscriptionLevelTexts(String level, int panels, int panelists) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "panels". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)